### DISCUSSION TITLE
Where did the settings go?

### DISCUSSION THREAD

**Discussion by SwimmingLink** [aclap-dev/video-downloadhelper#540] [Help and Q&A]
### Your question

Where did all of the settings go? I can't find where to state the formats/extensions to detect in the new version 9.

### Addon & CoApp version and details

version: 9.0.1.13
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\SwimmingLink\\dwhelper",
    "concurrent_downloads_max": 128,
    "theme": "dark",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": true,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 6
}

**Comment by paulrouget** [comment]
> I can't find where to state the formats/extensions to detect in the new version 9.

Normally, this was not necessary since version 8.0.
What kind of formats/extensions you want to change?

**Reply by SwimmingLink** [reply]
I've previously used this to get subtitle files as well, but they no longer show up (Haven't in the last version either)

**Reply by paulrouget** [reply]
I'm actually working on subtitles support at the moment. Hope to get that ready by v9.1.

**Reply by dexter7wolf** [reply]
please add the option to download also subtitles with videos in a separated file (srt, vtt o what you prefer!)
